Issues setting up the project locally

Hello, I'm trying to set up the project locally on my machine however I'm getting the following issues:

- When trying to run `make init`
```
bash ./compose/make_backend_env.sh ./.env
process_begin: CreateProcess(NULL, bash ./compose/make_backend_env.sh ./.env, ...) failed.
make (e=2): The system cannot find the file specified.
make: *** [Makefile:154: .env] Error 2
```

- When trying to run `docker-compose up`:
```
WARNING: The POSTGRES_PASSWORD variable is not set. Defaulting to a blank string.
ERROR: Couldn't find env file: C:\XXX\XXX\Desktop\zubhub\zubhub_backend\.env
```

### Environment:
- Windows 10
- GNU Make 4.3
- Docker Compose version 1.29.2
- Docker version 20.10.14

--- a/README.md
+++ b/README.md
@@ -175,7 +175,7 @@ Visit http://localhost:8000 on your browser to access the API documentation.
 
 - On your terminal/command line, navigate to **./zubhub/zubhub_frontend/zubhub** directory
 
-- Create a file named **.env** in the frontend root folder (same directory with package.json), then copy-past the following lines
+- Create a file named **.env** in the frontend root folder (same directory with package.json), with the following content
 
 ```sh
 #.env

--- a/README.md
+++ b/README.md
@@ -116,6 +116,8 @@ The copied URL will have the format of `https://github.com/<your github username
 
 ## Setup Backend
 
+> **_NOTE_**: For windows users, before running the commands below switch to bash (available through [WSL](https://docs.microsoft.com/en-us/windows/wsl/install)), you can achieve that by typing bash to your command line.
+
 - From the root of the repository, run:
 
 ```sh
@@ -149,8 +151,16 @@ To start:
 
 To stop:
 
+- Stop the running container
+
 ```sh
-       $ make stop or make down
+       $ make stop
+```
+
+- Stop & remove the running container
+
+```sh
+       $ make down
 ```
 
 You can run test by running `make test`.
@@ -163,15 +173,21 @@ Visit http://localhost:8000 on your browser to access the API documentation.
 
 ## Setup Frontend
 
-- Create a file named **.env** in the frontend root folder (same directory with package.json)
-- Copy the content of **.env.example** and paste into the new .env file.
 - On your terminal/command line, navigate to **./zubhub/zubhub_frontend/zubhub** directory
+
+- Create a file named **.env** in the frontend root folder (same directory with package.json), then copy-past the following lines
+
+```sh
+#.env
+REACT_APP_NODE_ENV=developement
+REACT_APP_BACKEND_DEVELOPMENT_URL=http://127.0.0.1:8000
+```
 
 ### **Using npm**
 
 This is advisable for a better development experience.
 
-- Run `npm install` to install the dependencies.
+- Run `npm install --legacy-peer-deps` to install the dependencies.
 - Run `npm start` to start the frontend.
 
 ### **Using the Docker Container**

